tests/browser/donations_payments/find_matched_gifts.robot

```diff
@@ -0,0 +1,86 @@
+*** Settings ***
+
+Resource        tests/NPSP.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+
+*** Test Cases ***
+
+Create Matching Donation
+    &{Org} =  API Create Organization Account    npsp__Matching_Gift_Company__c=true    npsp__Matching_Gift_Percent__c=100
```

Comment:
Using the npsp prefix will not work when NPSP is unmanaged. I need to create a keyword to get the namespace prefix for an object so we can put it in a variable and use that here. In the meantime, can we add `[tags]  unstable` to this test so that it won't break the automated builds?

Reply:
sure